### DISCUSSION TITLE
Compatible with attiny85

### DISCUSSION THREAD

**Discussion by Rudo909** [sandeepmistry/arduino-LoRa#498] [Q&A]
Hi everyone,
Is it possible to use Lora.h on a attiny85 and RFM95? It looks like it's not working now because the Attiny can not deal with SPI.h. 

It's working perfectly on my arduino and RFM95. 
Thanks a lot for the library 

Kind regards,

Ruud 
Thanks a lot

**Comment by IoTThinks** [comment]
Some similar issues here? https://github.com/sandeepmistry/arduino-LoRa/issues?q=attiny85

**Reply by Rudo909** [reply]
Thank you very much
